services/search/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# Use the official Python image.
+# https://hub.docker.com/_/python
+FROM python:3.7-slim
+
+# Copy local code to the container image.
+ENV APP_HOME /app
+ENV KEYWORDS_ENDPOINT="http://localhost:8081"
+WORKDIR $APP_HOME
+COPY . .
+
+# Install production dependencies.
+RUN pip install -r requirements.txt
+
+# Run the web service on container startup.
+CMD exec functions-framework --target=search_service
```

Comment:
Although this defaults to `PORT=8080`, maybe explicitly declaring it is good practice so that anyone who tries to spin-up the container in a local environment, knows which ports should get map.

Reply:
Another option, would be to leave a reference to `functions-framework` so that anyone can look into the library and understand how does it works ootb.